src/frameworks/svelte-kit.json

```diff
@@ -8,13 +8,13 @@
     "configFiles": []
   },
   "dev": {
-    "command": "svelte-kit dev",
-    "port": 3000,
+    "command": "vite dev",
+    "port": 5173,
     "pollingStrategies": [{ "name": "TCP" }, { "name": "HTTP" }]
   },
   "build": {
-    "command": "svelte-kit build",
-    "directory": "build"
+    "command": "vite build",
+    "directory": ".svelte-kit"
```

Comment:
Do we have to worry about sites with older versions of SvelteKit breaking with this change? I know we had a similar conversation with respect to `vite` but I think there might be a lot more users that this change would affect.

Reply:
I had thought about this, kind of like the PR #792, but in this case, these are the settings for the first official release for SvelteKit.

This also brings up the conversation again about potentially versioning framework information.

Reply:
This is something I mentioned to Nick[ in slack](https://netlify.slack.com/archives/CBC2U9MMG/p1664891406993729). There is a workaround by adding the `targetPort` to the `netlify,toml` file that can be added by anyone on an older version, but this will fix it so new updated sites don't have to do the workaround to get the CLI command to work.

Reply:
Got it, thanks both!